### ISSUE TITLE
[FirefoxNightlyWayland] Almost impossible to update on Arch: "Need more input"

### ISSUE THREAD

**Issue by glaubersm** [xhorak/firefox-devedition-flatpak#44]
Every time I need update Wayland build on Arch is a nightmare.
I get the same result almost always:

> Error: Failed to update org.mozilla.FirefoxNightlyWayland/x86_64/master: While pulling app/org.mozilla.FirefoxNightlyWayland/x86_64/master from remote org.mozilla.FirefoxNightlyWayland-2-origin: Need more input
error: One or more operations failed

**Comment by xhorak** [comment]
Sorry, we've run of space on the server again, so there could be some files missing. It should be fine now.

**Comment by jsparber** [comment]
I still have the same problem

**Comment by hungrymonkey** [comment]
same here on kde neon user edition

```
 flatpak install org.mozilla.FirefoxRepo  org.mozilla.FirefoxDevEdition
Updating: org.gnome.Platform/x86_64/3.24 from gnome
No updates.
Updating: org.gnome.Platform.Locale/x86_64/3.24 from gnome
No updates.
Installing: org.mozilla.FirefoxDevEdition/x86_64/master from org.mozilla.FirefoxRepo

Writing objects: 1                                                                                                                                                                         
error: While pulling app/org.mozilla.FirefoxDevEdition/x86_64/master from remote org.mozilla.FirefoxRepo: Need more input
```

**Comment by Hapoofesgeli** [comment]
Still getting the same error on Arch. Tried updating more than 20 times and all failed, the only way is remote downloading the bundle, since even simply downloading it will fail at some point.

**Comment by kwill** [comment]
Can this be closed as a duplicate of #1 ?

**Comment by xhorak** [comment]
Yes, it's the very #1.
